.github/workflows/cicd.yaml

```diff
@@ -0,0 +1,93 @@
+name: CI/CD
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  ci:
+    name: CI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        stable: false
+        go-version: 1.16.0-beta1
```

Comment:
probably best to use 1.15 unless you need to embed files - the real 1.16 will be out soon enough anyway. If you do change this to 1.15, you'll also need to remove the `stable: false` line above. I note you use 1.15 in the Dockerfile.

Reply:
done.

Reply:
You have not removed the `stable: false` line. That might be why the CI is failing (haven't look yet, but I see the red cross)